src/tools/dev/scripts/bv_support/bv_glfw.sh

```diff
@@ -0,0 +1,176 @@
+function bv_glfw_initialize
```

Comment:
Looks like a wholly new module. I think this should be put in a sep. PR.

Reply:
I agree with @markcmiller86 
The build_visit cleanup should be separate from the addition of new modules or updates to current module versions.
I checked out your branch and tried to run build_visit from it, but ran into a lot of problems with the new library versions not being downloadable.  Makes it hard to test the cleanup changes for problems.